.github/workflows/tests.yml

```
@@ -43,5 +43,5 @@ jobs:
       run: npm ci
       working-directory: ./phone-store
     - name: Run e2e tests
-      run: npm run e2eci
+      run: ng e2e --headless
```

You'll notice it failed. This is why I've been using `npm run` scripts.

---

```suggestion
      run: npm run e2e
```
I think when on a headless system it is automatically headless.